### ISSUE TITLE
Number.MAX_VALUE for `minimum` or `maximum` generates null

### ISSUE THREAD

**Issue by lukasz-kuzynski-11sigma** [json-schema-faker/json-schema-faker#679]
Using Number.MAX_VALUE in `minimum` or `maximum` for number type makes jsf to generate `null` instead number.

**Schema**

```json
{
  "type": "object",
  "properties": {
    "double": {
      "type": "number",
      "minimum": -1.7976931348623157e+308,
      "maximum": 1.7976931348623157e+308
    }
  }
}
```

**Result**
```
{
  "double": null
}
```

**Expected result**
```
{
  "double": 1001 // or any number
}
```

**Comment by lukasz-kuzynski-11sigma** [comment]
Fixed

**Comment by lukasz-kuzynski-11sigma** [comment]
Is it possible to release this fix under version 0.4.* instead of waiting for whole 0.5.0 to be release? If not can we please release RC with that fix?
